variables.csv

```diff
@@ -0,0 +1,200 @@
+HUIDEMO,HUI Emotion,Emotional problems - function code - (D),Health status,Health utility index,Categorical,N/A,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::HUIADEMO, cchs2003_p::HUICDEMO, cchs2005_p::HUIEDEMO, [HUIDEMO]",
+HUIDHSI,Overall HUI,Health Utility Index (HUI3) - (D),Health status,Health utility index,Continuous,N/A,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::HUIADHSI, cchs2003_p::HUICDHSI, cchs2005_p::HUIEDHSI, [HUIDHSI]",
+HUIGDEX,HUI Dexterity,"Dexterity trouble - function code (D, G)",Health status,Health utility index,Categorical,N/A,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::HUIAGDEX, cchs2003_p::HUICGDEX, cchs2005_p::HUIEGDEX, [HUIGDEX]",
+HUIGHER,HUI Hearing,"Hearing problems - function code (D, G)",Health status,Health utility index,Categorical,N/A,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::HUIAGHER, cchs2003_p::HUICGHER, cchs2005_p::HUIEGHER, [HUIGHER]",
+HUIGMOB,HUI Mobility,"Mobility trouble - function code (D, G)",Health status,Health utility index,Categorical,N/A,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::HUIAGMOB, cchs2003_p::HUICGMOB, cchs2005_p::HUIEGMOB, [HUIGMOB]",
+HUIGSPE,HUI Speech,"Speech trouble - function code - (D, G)",Health status,Health utility index,Categorical,N/A,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::HUIAGSPE, cchs2003_p::HUICGSPE, cchs2005_p::HUIEGSPE, [HUIGSPE]",
+HUIGVIS,HUI Vision,"Vision trouble - function code - (D, G)",Health status,Health utility index,Categorical,N/A,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::HUIAGVIS, cchs2003_p::HUICGVIS, cchs2005_p::HUIEGVIS, [HUIGVIS]",
+HUPDPAD,HUI Pain,Act. prevent/pain - function code - (D),Health status,Health utility index,Categorical,N/A,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2012_p, cchs2013_2014_p, cchs2014_p","cchs2001_p::HUIADPAD, cchs2003_p::HUICDPAD, cchs2005_p::HUIEDPAD, [HUPDPAD]",
+HWTGBMI,BMI,"BMI / self-report - (D,G)",Health status,BMI,Continuous,kg/m2,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p","cchs2001_p::HWTAGBMI, cchs2003_p::HWTCGBMI, cchs2005_p::HWTEGBMI, cchs2015_2016_p::HWTDGCOR, cchs2017_2018_p::HWTDGBMI, [HWTGBMI]",
```

Comment:
Can you explain why you chose `HWTDGCOR`? I believe `HWTDGBCC` is the correct variable. Regardless, there are quite a few BMI variables in the CCHS 2015-16 that are different from other cycles. Let's be careful. 

Would you mind reviewing the BMI variables in 2015 and providing documentation or a description about what the different variable are designed and added to this cycle?

Reply:
I don't think we currently have adjusted BMI in cchsflow. @wyusuf068 do you know? If we don't have adjusted BMI, we should add as an issue for future addition to cchsflow. 

https://www150.statcan.gc.ca/n1/pub/82-003-x/2008002/article/10569/5202509-eng.htm

https://www.halton.ca/Document/Health-Statistics/Adjusted-Body-Mass-Index-(BMI)/Body-Mass-Index-BMI

https://www.halton.ca/Document/Health-Statistics/Adjusted-Body-Mass-Index-(BMI)/Body-Mass-Index-BMI

https://pubmed.ncbi.nlm.nih.gov/22106788/

Reply:
@abadereddeen What measure of BMI did you recently use in the CCHS?

Reply:
`HWTDGCOR` was used because the adjusted BMI is a continuous BMI variable. `HWTDGBCC` is a categorical BMI variable for 2015-2016 (underweight, normal weight, overweight, obese). There weren't any other self-reported continuous BMI variables for all age groups in 2015-2016. It would be safe to assume that continuous BMI is not present for 2015.

Reply:
We will need to calculate continuous BMI from height and weight if there isn't a measure. I am surprised there isn't a continuous measure. You can check the details of `HWTDGCOR` to see how that was derived but it seems at first glance the variable is adjusted for self-report bias for height and weight.

Reply:
I found the derived variable documentation. It looks like `HWTDGBMI` may be what we want. Can you review? The documentation also has the formulas for `HWTDGCOR` - let's discuss later about adding. 

[CCHS_2015_2016_PUMF_Derived_Variables.pdf](https://github.com/Big-Life-Lab/cchsflow/files/7165246/CCHS_2015_2016_PUMF_Derived_Variables.pdf)

Reply:
@DougManuel at the moment we don't have adjusted BMI. I think this is something we should include for all cycles as a derivation of `BMI_der`. Will need to look into how they calculated the adjustment weights and see if they can be used for earlier cycles.

`HWTDGBMI` would be the equivalent of `HWTGBMI` in earlier cycles. Self-reported Height and weight variables are in the same units as earlier cycles so they can be included and fed into `BMI_der`, which @kittychenn has done 👍

Reply:
@kittychenn and I added an issue for corrected BMI. https://github.com/Big-Life-Lab/cchsflow/issues/102 The formula is in the PDF above.